src/module.ml

```diff
@@ -186,3 +207,8 @@ let wrapped_compat t =
 module Name_map = struct
   type nonrec t = t Name.Map.t
 end
+
+let is_public t = Visibility.is_public t.visibility
+
+let set_private t =
+  { t with visibility = Visibility.Private }
```

Comment:
We shouldn't need the module path here

Reply:
What is the module path?

Reply:
`Visibility.`

Reply:
Ah, fixed. I thought you mean some path value of Module.t 😅